src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -209,6 +209,18 @@ void AddSection(string kind, ImmutableArray<TaggedText> taggedParts)
                 AddSection(QuickInfoSectionKinds.DocumentationComments, documentationContent);
             }
 
+            var returnsDocumentationContent = GetReturnsDocumentationContent(symbols, groups, semanticModel, token, formatter, syntaxFactsService, cancellationToken);
+            if (!returnsDocumentationContent.IsDefaultOrEmpty)
+            {
+                var builder = ImmutableArray.CreateBuilder<TaggedText>();
+                builder.AddLineBreak();
```

Comment:
The content starts with a line break...?

Reply:
Yes, this is what separates it from the preceding text. The same occurs for exceptions text:

https://github.com/dotnet/roslyn/blob/5d247289d30d183ecbfaa7f59163ea33b915a36d/src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs#L162-L164

Reply:
Yes, this is what separates it from the preceding text. The same occurs for exceptions text:

https://github.com/dotnet/roslyn/blob/5d247289d30d183ecbfaa7f59163ea33b915a36d/src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs#L162-L164